R/plots.R

```diff
@@ -1,12 +1,18 @@
 create_main_projection_chart <- function(dat, horizon) {
-  conv_lab <- \(x) {
+  convert_year_labels <- function(x) {
     paste0(format(x, "%Y"), "/", (as.integer(format(x, "%Y")) + 1) %% 100)
   }
-  fy_as_date <- \(x) as.Date(sub("_[0-9]{2}$", "-01-01", x))
+  fy_as_date <- \(x) as.Date(sub("_[0-9]{2}$", "-04-01", x))
 
   full_dat <- dplyr::mutate(dat, dplyr::across("fin_year", fy_as_date))
   filtered_dat <- dplyr::filter(full_dat, .data$fin_year <= fy_as_date(horizon))
 
+  label_fn <- if (max(full_dat$projected_count) < 1e6) {
```

Comment:
Possibly contentious: switch scaling and labelling of y-axis depending if we are mostly working in hundred-thousands or millions

Reply:
I think it's fine here